test/specs/create-journey-tests.e2e.js

```diff
@@ -120,11 +140,61 @@ describe('Create journey tests', () => {
         'infrastructure'
       ]) {
         await $(
-          `[data-testid="${statusTag}-status-tag"]*=Success`
+          '[data-testid="' + statusTag + '-status-tag"]*=Success'
```

Comment:
nit: could just use template string `[data-testid="${statusTag}-status-tag"]*=Success`

Reply:
That's what it was. And lint blew up.